src/js/components/AppListComponent.jsx

```diff
@@ -196,6 +196,18 @@ var AppListComponent = React.createClass({
       .sortBy((app) => {
         return app[sortKey];
       }, state.sortDescending)
+
```

Comment:
Not sure I like the extra lines around this block. Can't we just move the comment inside the `sort` ?